core/src/main/java/org/infinispan/commands/control/LockControlCommand.java

```diff
@@ -78,6 +81,11 @@ public LockControlCommand(Object key, ByteString cacheName, long flags, GlobalTr
       this.globalTx = gtx;
    }
 
+   public LockControlCommand(ByteString cacheName, GlobalTransaction gtx) {
+      super(cacheName);
+      unlock = true;
```

Comment:
Please make this a parameter, it's too surprising IMO if `unlock` has one default value with some constructors and another default value with other(s).